CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- ADD: allow devices with the same device_id in the same service and subservice but different apikey (#1589)
```

Comment:
Should documentation be updated (or created?) taking into account this PR?

@mapedraza what do you think?

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1592/commits/ce50d89456c6b034d988c8a951e09cd0f8ae2686

Reply:
```suggestion
- Add: allow devices with the same device_id in the same service and subservice but different apikey (#1589)
```